automation/taskcluster/lib/tasks.py

```diff
@@ -303,7 +345,6 @@ def _craft_default_task_definition(
             "dependencies": [self.task_id] + dependencies,
             "requires": "all-completed",
             "routes": routes + [
-                "tc-treeherder.v2.reference-browser.{}".format(self.commit)
```

Comment:
Thanks for fixing the route on forks. Let's make the solution generic so it works on both the main repo and forks.

How about we do:

```py
if self.trust_level == 3:
    routes.append("tc-treeherder.v2.reference-browser.{}".format(self.commit))

return {
    "provisionerId": provisioner_id,
    # ...
    "routes": routes,
    # ...
}

```

?

Reply:
done!